src/Item.js

```diff
@@ -83,6 +142,25 @@ function Item({ userToken, item }) {
         />
         {itemName}
       </label>
+      {/* // [delete-item] 1. Create a button next to each item in list */}
+      <button type="button" onClick={handleOpenModal}>
+        X
+      </button>
+      {/* // [delete-item] 2. Create a button modal that will render when user tries to delete item */}
+      <Modal
+        open={openModal}
+        onClose={handleCloseModal}
+        aria-labelledby="simple-modal-title"
+        aria-describedby="simple-modal-description"
```

Comment:
You don't have a modal description, so doesn't look like you need the `aria-describedby`